content/docs/glossary/clawback.mdx

```diff
@@ -0,0 +1,297 @@
+---
+title: Clawbacks
+order:
+---
+
+import { CodeExample } from "components/CodeExample";
+import { Alert } from "components/Alert";
+
+Protocol 17 introduces operations that allow asset issuers to maintain tighter control over how their asset is distributed to the world. Specifically, it gives them power to **burn their asset** after a [payment][ex1] or [claimable balance][ex2] to another account, effectively removing it from the recipient's balance sheet:
```

Comment:
I would write the last sentence differently to `Specifically, it gives them power to **burn their asset** from a trustline or claimable balance, effectively removing it from the recipient's balance sheet.` It's possible for an account to fund it's trustline using offers (and I _think_ this is a common way to fund trustlines?) instead of using payments. I think it's more relevant to talk about trustlines here, but please let me know if I'm thinking too low level here.

Reply:
Good call, covering all of the ways that accounts can have funds is hard. 8f9e286